### PR TITLE
fix bug with start having ! in front

### LINKED ISSUE #348: iis_pool :start fails to start app pool if runtime_version not set
Note: I've focussed specifically on the `:start` action here, but I think there are several places in _pool.rb_ where `current_resource.runtime_version` checks need to be removed or fixed.

### Cookbook version
6.4.0

### Chef-client version
Chef Client 13.0.118 (Kitchen 1.16.0)

### Platform Details
Windows Server 2012 R2

### Scenario:
Start an app pool with:

```
iis_pool app_pool_name do
  action :start
end
```

### Steps to Reproduce:
Run the `:start` action on an app pool that is in the stopped state. Note: The `runtime_version` option must not be provided.

### Expected Result:
The app pool ends up in the started state.

### Actual Result:
The app pool is still in the stopped state.

If debug messages are enabled, the debug log shows the following message:

> DEBUG: iis_pool[AppPoolName] already running - nothing to do

### PR DESCRIPTION
**Pull request by EasyAsABC123**
also added current_resource for good measure
Signed-off-by: Justin Schuhmann <jmschu02@gmail.com>

### Description

This was a simple typo on the start property, also added current_resource to make sure the checks worked as intended, new test added to verify issue.

### Issues Resolved

#348 

### Check List

- [x] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

**Comment by EasyAsABC123** [comment]
@iennae it is only used internally and shouldn't be passed.  I suppose i could make it not a property and instead a `@running` variable but i thought that seemed dirty.  it is equivalent to `exists` in that they should never be passed and would be ignored if passed

File: resources/pool.rb [hunk 1]
```diff
@@ -80,7 +80,7 @@
 property :smp_processor_affinity_mask, Float, default: 4_294_967_295.0, coerce: proc { |v| v.to_f }
 property :smp_processor_affinity_mask_2, Float, default: 4_294_967_295.0, coerce: proc { |v| v.to_f }
 
-property :running, [true, false], desired_state: true
+property :running, [true, false]
```
Comment (smurawski) [inline resources/pool.rb]: Why remove the `desired_state: true` from the `running` property?  It's not a settable property by the user right?

Comment (EasyAsABC123) [inline resources/pool.rb]: @smurawski correct it isn't

Comment (EasyAsABC123) [inline resources/pool.rb]: but `desired state` looks like http://www.rubydoc.info/gems/chef/Chef%2FProperty%3Adesired_state%3F

Comment (smurawski) [inline resources/pool.rb]: @EasyAsABC123 My reading of https://docs.chef.io/custom_resources.html#desired-state is that `desired_state: true` means that the property will get its value from the current state of the system (and not via the instance of the resource in the recipe).

Comment (EasyAsABC123) [inline resources/pool.rb]: @smurawski that's awesome, definitely not how i read it ;) but i'm glad that the property to make a property private exists, thanks

**Pull request merged**